apps/system/js/multi_screen_controller.js

```diff
@@ -239,6 +195,11 @@
         data.type + ', ' + JSON.stringify(data.detail));
 
       switch(data.type) {
+        case 'remote-system-ready':
+          this.postMessage(this.screenId,
+                           'launch-presentation-app',
+                           this.requestConfig);
```

Comment:
Should we clear `screenId` (`requestDeviceId`) and `requestConfig` after it has been delivered?